app/components/object-detail-widget/component.js

```diff
@@ -15,6 +15,25 @@ export default Ember.Component.extend({
         return JSON.stringify(data, null, '    ');
     }),
 
+    identifiers: Ember.computed('objectData._source.identifiers', function() {
+        var data = this.get('objectData');
+        var identifiers = data._source.identifiers;
+        var linkIds = [];
+        var otherIds = [];
+        identifiers.forEach(function(id) {
+            if (id.includes('http')) {
```

Comment:
Includes will match anywhere in the string, `thisaintnourIhttp://`. That also that won't match `https`.